django_app/redbox_app/redbox_core/models.py

```diff
@@ -96,7 +96,7 @@ def url(self) -> URL:
 
     @property
     def name(self) -> str:
-        return self.original_file_name if self.original_file_name else self.original_file.name
+        return self.original_file.file.name  # handles duplicate names
```

Comment:
Do we need the extra `.file.` nest here?

Does this mean in the documents page the users see the documents name with the hash on the end?

Reply:
`original_file.file.name` gives the name with the hash, `original_file.name` give the name without the hash. 

We need the first for core-api so that the embeddings are different (in case files have the same name but different content). But I agree we can use original_file.name for the user, will update this now.

Reply:
Ok that sounds ideal, if that's the case - that we can get both hashed and unhashed name from the file - then we probably don't need the property on the File model and can change the documents views to reflect that

Reply:
(unless we want to make it more readable in usages, in which case I'm fine with the two properties)